packages/jaeger-ui/src/components/TracePage/TracePageSearchBar.css

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.TracePageSearchBar {
+  max-width: 20rem;
+}
+
+.TracePageSearchBar--bar {
+  display: flex;
+  max-width: 20rem;
+}
+
+.TracePageSearchBar--count {
+  opacity: 0.6;
+}
+
+.TracePageSearchBar--btn {
+  transition: 0.5s;
+}
+
+.TracePageSearchBar--btn-hide {
```

Comment:
For consistency, please change the selector to something like `.TracePageSearchBar--btn.is-*` where `*` describes the state, e.g. `.DiffNode--labelCell.is-added`.